docs/guides/esp32/build_app_and_commission.md

```diff
@@ -190,6 +191,16 @@ $ out/debug/chip-tool pairing ble-wifi 12345 MY_SSID MY_PASSWORD 20202021 3840
      $ ./out/debug/chip-tool pairing ble-thread 12345 hex:<operational-dataset> 20202021 3840
     ```
 
+#### Commissioning the Ethernet device (ESP32-Ethernet-Kit)
+
+```
+$ out/debug/chip-tool pairing ethernet 12345 20202021 3840 device-remote-ip device-remote-port
```

Comment:
Replace device-remote-port by the actual value?

Reply:
ok